### ISSUE TITLE
Bug when filling the captcha

### ISSUE THREAD

**Issue by trapextremo** [Jessecar96/SteamDesktopAuthenticator#432]
Everytime that I fill the captcha, it is ok, but it appears again. I do the captcha correctly but then it appears again and again so I can't add my steam account.

**Comment by free020603** [comment]
I also have same problem with you man.... did you fix the problem yet? My steam name 'An Innocent Cop.' add me. I wanna discuss with you

**Comment by free020603** [comment]
I also tried import from Android. My Android phone is rooted and everything was set up perfectly. it was extracting file 1/2. and that is it. after showing up "extracting file 1/2". it doesn't work. I've wait for 5 hours. still not working. what is wrong with this program. I also tried login new. after I entered my correct ID and PW. it kept showing up CATCHA code. fucking CAPTCHA code. So, I tried 47 times to enter correct CAPTCHA code. my spell and special key was fine. It just keeps show up. fucking endless CAPTCHA. Also, I know Bronie guy keep closing every issues about endless CAPTCHA. What the fuck is wrong with this program
